packages/flutter_tools/lib/src/flutter_plugins.dart

```diff
@@ -558,6 +558,7 @@ const String _dartPluginRegistryTemplate = '''
 // ignore_for_file: directives_ordering
 // ignore_for_file: lines_longer_than_80_chars
 // ignore_for_file: depend_on_referenced_packages
+// ignore_for_file: prefer_final_parameters
```

Comment:
Instead of adding this ignore let's just fix the actual warning by making `Registrar registrar` final in line 570.